optuna/storages/_rdb/alembic/versions/v2.4.0.a.py

```diff
@@ -0,0 +1,194 @@
+"""empty message
+
+Revision ID: v2.4.0.a
+Revises: v1.3.0.a
+Create Date: 2020-11-17 02:16:16.536171
+
+"""
+from typing import Any
+
+from alembic import op
+import sqlalchemy as sa
+
+from sqlalchemy.exc import SQLAlchemyError
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import orm
+
+from optuna.study import StudyDirection
+
+
+# revision identifiers, used by Alembic.
+revision = "v2.4.0.a"
```

Comment:
We should probably clarify what changes to include together in a release. I believe we at least want the storage interface changes and ideally the entire MO refactoring (up to samplers, excluding pruning) together in order to have a clean release note and to avoid the risk of having to reiterate on the schema between multiple releases. We're currently planning for code freeze Dec. 4 so v2.5 might be safer. Being a bit flexible with v2.4 having it delayed is another option.

Reply:
[Note] In internal discussions, it was decided to delay the release of v2.4 to complete refactoring of the Multi-objective module as much as possible.